pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/JsonIndexTest.java

```diff
@@ -371,6 +372,40 @@ public void testGetValuesForKeyAndDocs()
     }
   }
 
+  @Test
+  public void testSkipInvalidJson() throws Exception {
+    JsonIndexConfig jsonIndexConfig = new JsonIndexConfig();
+    jsonIndexConfig.setSkipInvalidJson(true);
```

Comment:
can you 
1. add a test when it is false it still throws? 
2. add a test for other exception is still throws when this is set to true?

Reply:
sure

Reply:
> can you
> 
> 1. add a test when it is false it still throws?
> 2. add a test for other exception is still throws when this is set to true?

for the 2nd requirement, I haven't figure a way to test since the JsonUtils make the objectReader as a static final member.
```
  public static final ObjectReader DEFAULT_READER = DEFAULT_MAPPER.reader();
```

Reply:
sorry for the late reply. that fine if we can't find it.